src/app/partials/nav/header-static/header-static.component.html

```diff
@@ -1,25 +1,39 @@
-<header id="header-static">
-  <ul class="inline-list left-list left">
-    <li class="logo">
-      <img src="/assets/images/EvalAI_favicon.png" alt="">
-    </li>
-  </ul>
-  <ul class="inline-list right-list right right-nav">
-    <li>
-      <a class="light-link" href="/"> Challenges</a>
-    </li>
-    <li>
-      <a class="light-link" href="/"> News</a>
-    </li>
-    <li>
-      <a class="light-link" href="/"> Forum</a>
-    </li>
-    <li>
-      <a class="light-link" href="/"> Join Us</a>
-    </li>
-    <li class="pointer">
-      ...
-    </li>
-  </ul>
-  <div class="clear-fix"></div>
+<header id="header-static" [class.header-shadow]="headerWhite" [class.header-shadow-large]="scrolledState" [class.white-bg]="headerWhite" [class.dark-bg]="isMenuExpanded && atHome && innerWidth <= 810">
+    <div class="logo-container">
+      <img src="assets/images/EvalAI_favicon.png" class = "logo-header" [class.invert-color]="headerWhite" *ngIf="atHome" (click)="isMenuExpanded = false; navigateTo(['']);"/>
```

Comment:
Why are we not rapping it in route link `a` instead of navigateTo? @live-wire any specific reason.